marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/installer/command/AbstractInstallerCommand.java

```diff
@@ -81,4 +84,21 @@ protected String getServerMajorVersion() {
             return "9";
         }
     }
+
+    protected ObjectNode canInstallDhs(String installedHubVersion, MarkLogicVersion mlVersion){
```

Comment:
Since this class has a reference to HubConfig, I don't think it needs any inputs here, and it should do the work of using Versions to get the needed data. That will simplify what the subclass has to do.

Reply:
The only reason I had the 2 parameters for this method is for using it in the CanInstallDhsCommandTest. Without the 2 params, I can't test this method for various inputs.

Reply:
Ah, that's a good reason. I think the way to go then is keep this method, and add a protected method above it that calls it and that the subclasses use. That eliminates the duplication in the subclasses.